File: app/src/main/java/com/raju/javabaseproject/data/source/local/AppDatabase.java
package com.raju.javabaseproject.data.source.local;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.raju.javabaseproject.data.model.User;
import com.raju.javabaseproject.data.source.local.dao.UserDao;

@Database(entities = {User.class}, version = AppDatabase.VERSION)
public abstract class AppDatabase extends RoomDatabase {
    static final int VERSION = 1;
    public abstract UserDao userDao();
}

File: app/src/main/java/com/raju/javabaseproject/data/source/local/AppDatabase.kt
package com.raju.javabaseproject.data.source.local

import androidx.room.Database
import androidx.room.RoomDatabase
import com.raju.javabaseproject.data.model.User
import com.raju.javabaseproject.data.source.local.dao.UserDao

@Database(entities = [User::class], version = AppDatabase.VERSION)
abstract class AppDatabase : RoomDatabase() {
    abstract fun userDao(): UserDao

    companion object {
        const val VERSION = 1
    }
}